src/pages/location-message/location-message.html

```diff
@@ -0,0 +1,22 @@
+<ion-header>
+  <ion-toolbar color="whatsapp">
+    <ion-title>Send Location</ion-title>
+
+    <ion-buttons end>
+      <button ion-button class="dismiss-button" (click)="viewCtrl.dismiss()"><ion-icon name="close"></ion-icon></button>
+    </ion-buttons>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content class="location-message">
+  <ion-list>
```

Comment:
the rationale behind this is that it would be easier to look at, and the view would be more organized. If you'r convinced that it's unnecessary, I'm with you